nemo/collections/asr/parts/submodules/jasper.py

```diff
@@ -462,17 +462,21 @@ def __init__(
         self.change_context_window(context_window=context_window)
 
         # Set default max sequence length
-        self.set_max_len(1)
+        self.set_max_len(16)
 
     def forward(self, x, lengths):
         return self.forward_for_export(x, lengths)
 
     def forward_for_export(self, x, lengths):
         # The use of negative indices on the transpose allow for expanded SqueezeExcite
+        max_len = x.shape[-1]
+        if max_len > self.max_len:
+            self.set_max_len(max_len)
```

Comment:
Is this fine inside of export? Wont the calculation of new mask affect ONNX?

Reply:
This will be traced and if() condition will be false, so no-op.